### DISCUSSION TITLE
Issues getting Heroku Credits under the Github Student developers pack.

### DISCUSSION THREAD

**Discussion by GrantWils23** [community/community#39953] [GitHub Education]
### Select Topic Area

Question

### Body

Hi there, I'm having issues with getting my GitHub student developers pack to register for credits with Heroku. It keeps me going around in one vicious circle. has anyone had an issue like this? i dont know what end i have an issue with. Github or Heroku. Heroku says it cant verify my account. any ideas?

**Comment by ossanna16** [comment]
Hi there @GrantWils23 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
